src/Generate/ViewForm.php

```diff
@@ -178,6 +225,7 @@ protected function buildCreate() {
             'modelJSName' => $this->modelJSName,
             'modelLangFormat' => $this->modelLangFormat,
             'resource' => $this->resource,
+            'containsPublishAtColumn' =>  in_array("published_at", array_column($this->readColumnsFromTable($this->tableName)->toArray(), 'name')),
```

Comment:
Rename to two column layout or something different and also create if createdByAdminUser/updatedByAdminUser is present